client/src/components/Library.jsx

```diff
@@ -223,7 +223,10 @@ class Library extends Component {
 
 
     return (
-      <div className="Library">
+      <div
+        className="Library"
+        data-test-id="library"
```

Comment:
why data-test-id and not just id?

Reply:
Hi Dan, the reasons are:
1 - In HTML 5 there is a particular kind of attributes called data attributes:
          https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes
2 - By agreement, we use these attribute names to call the attributes we use for tests.
3 - Those attributes are useful only when we do the test, so it would be interesting to delete or hide them when we build, and there is an option that allow us to do it.